.github/workflows/test.yaml

```diff
@@ -24,15 +24,7 @@ jobs:
     strategy:
```

Comment:
Need to change the above line to 
```diff
-     uses: pion/.goassets/.github/workflows/test.reusable.yml@master
+     uses: stv0g/.goassets/.github/workflows/test.reusable.yml@more-platforms
```
to actually test the changes in https://github.com/pion/.goassets/pull/166